src/gsModeling/gsSurfaceReparameterization.h

```diff
@@ -17,221 +17,242 @@ Author(s): Ye Ji
 
 #include <gsHLBFGS/gsHLBFGS.h>
 
-namespace gismo
-{
+namespace gismo {
 
-template<short_t d=2, typename T=real_t>
-class gsObjFuncSurface : public gsOptProblem<T>
-{
+template <short_t d = 2, typename T = real_t>
+class gsObjFuncSurface : public gsOptProblem<T> {
```

Comment:
Please put the `{` on a new line, also for all the other functions, constructors and loops (of more than one line). This makes the code more readable